css/style.css

```diff
@@ -25,4 +25,4 @@ input[type=text],input[type=email],input[type=number], select, textarea, button{
 }
 button{
   background-color: #38ac64;
-}
```

Comment:
your work is excellent but there must be one empty line at the end of CSS file. so I suggest to include one blank line at the end of file.